docs/maui/layouts/index.md

```diff
@@ -14,4 +14,5 @@ The .NET MAUI Community Toolkit extends the list of .NET MAUI layouts. Here are
 | View | Description |
 | --------- | ----------- |
 | [`StateContainer`](StateContainer.md) | The `StateContainer` bindable properties enable any Layout derived element to become state-aware. |
-| [`UniformItemsLayout`](UniformItemsLayout.md) | The `UniformItemsLayout` is a layout where all rows and columns have the same size. |
\ No newline at end of file
+| [`UniformItemsLayout`](UniformItemsLayout.md) | The `UniformItemsLayout` is a layout where all rows and columns have the same size. |
+| [`DockLayout`](DockLayout.md) | The `DockLayout` is a layout where views can be docked to the sides (top, left, right, bottom) of the layout container. |
```

Comment:
I like to order this list alphabetically, can we put this above `StateContainer`? Annoyingly GitHub won't let me provide a suggestion

Reply:
Yep. Done.